src/templates/policies.bicep

```diff
@@ -248,7 +248,7 @@ var policyDefinitionsByCloudType = {
 var policyDefinitions = concat(policyDefinitionsByCloudType.All, policyDefinitionsByCloudType[cloudEnv])
 
 // Create the Policy Definitions as needed for the target cloud environment
-resource PolicyDefinitions 'Microsoft.Authorization/policyDefinitions@2020-09-01' = [for policy in policyDefinitions: {
+resource PolicyDefinitions 'Microsoft.Authorization/policyDefinitions@2023-04-01' = [for policy in policyDefinitions: {
   name: policy.name
   properties: {
```

Comment:
do you need to add version into the properties here too?

Reply:
We're only using this for our custom policies, which doesn't support versioning, so not needed. Version pinning needs to be done in assignments for built-in policies and initiatives and in our custom initiatives referencing built-in policies.